plugin-flex-ts-template-v2/src/feature-library/admin-ui/utils/helpers.ts

```diff
@@ -7,6 +7,11 @@ import AdminUiService from './AdminUiService';
 
 const acronyms = ['id', 'ui', 'sip', 'pstn', 'sms', 'crm', 'sla', 'cbm', 'url'];
 const hiddenFeatures = ['admin_ui'];
+const docsBaseUrl = 'https://twilio-professional-services.github.io/flex-project-template';
```

Comment:
nothing i think we need to address right now but later down the road we might want to think about doing something here that might dynamically link this to the repo it belongs to rather than the base template docs along with all the other admin panel docs links

Reply:
Maybe it could be configurable, not sure how we could automatically deduce this reliably.